CMakeLists.txt

```diff
@@ -9,7 +9,7 @@
 #################################################################################
 # General settings
 #################################################################################
-cmake_minimum_required(VERSION 2.8.3 FATAL_ERROR)
+cmake_minimum_required(VERSION 3.1 FATAL_ERROR)
```

Comment:
As per RFC3 this can go all the way to 3.5. No need to raise it that high unless there's a reason for it but if it makes things easier for you here please go ahead.

Reply:
Done